### PR TITLE
R21.0 link consolidation

### PR DESCRIPTION
**Pull request by dougkenik**
Created a class which holds all of the links for SmartSlice, and registered a QML type so any UI component can access it. Modified all of our code so it uses the new class.

### CODE REVIEW

**Comment by dougkenik** [comment]
Also added the work for #491, since it is closely tied to this

**Review by colman-hartinger — APPROVED** [review]
looks and runs good for me

File: SmartSlicePlugin/SmartSliceAbout.qml [hunk 1]
```diff
@@ -105,7 +108,10 @@ UM.Dialog {
                         anchors.fill: parent
                         hoverEnabled: true
                         cursorShape: Qt.PointingHandCursor
-                        onClicked: Qt.openUrlExternally('https://help.tetonsim.com/eula-end-user-license-agreement')
+                        onClicked: {
+                            console.log("EULA: ", aboutDialog.urlHandler.eula)
```
Comment (bradymadams) [inline SmartSlicePlugin/SmartSliceAbout.qml:112]: Do we want this log?

File: SmartSlicePlugin/SmartSliceURL.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+from .stage.SmartSliceStage import SmartSliceStage
+
+class URLHandler(QObject):
+    def __init__(self, parent=None):
+        super().__init__()
+
+        self._home = 'https://tetonsim.com'
+        self._help = 'https://help.tetonsim.com'
+        self._help_email = 'mailto:help@tetonsim.com'
```
Comment (bradymadams) [inline SmartSlicePlugin/SmartSliceURL.py:12]: Double quote strings. Same for all below strings in this file.

**Review by bradymadams — CHANGES_REQUESTED** [review]

**Review by bradymadams — APPROVED** [review]

**Pull request merged**